client/v3/maintenance.go

```diff
@@ -244,18 +244,19 @@ func (m *maintenance) SnapshotWithVersion(ctx context.Context) (*SnapshotRespons
 	}
 	go func() {
 		// Saving response is blocking
-		err = m.save(resp, pw)
+		err := m.save(resp, pw)
 		if err != nil {
 			m.logAndCloseWithError(err, pw)
 			return
 		}
 		for {
-			resp, err := ss.Recv()
+			sresp, err := ss.Recv()
```

Comment:
use new `sresp` here because the old name shadows the L240.

./maintenance.go:253:4: declaration of "resp" shadows declaration at line 240